build.gradle

```diff
@@ -303,12 +262,61 @@ configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driv
 }
 
 task docs(type: Javadoc) {
-    source subprojects.grep({ it.name != 'util' }).collect {project -> project.sourceSets.main.allJava }
-//    options = subprojects.first().javadoc.options
```

Comment:
Options became read only in the newer (4.0?) version of gradle - so to change it you have to use the merger method.